pom.xml

```diff
@@ -199,6 +199,23 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>knative</id>
+            <activation>
+                <property>
+                    <name>knative</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.container-image.image>${env.KO_DOCKER_REPO}/${project.artifactId}:${env.TAG}</quarkus.container-image.image>
+
+                <quarkus.container-image.build>true</quarkus.container-image.build>
+                <quarkus.container-image.push>false</quarkus.container-image.push>
+
+                <!-- Unset to prevent default inheritance -->
+                <quarkus.container-image.insecure>false</quarkus.container-image.insecure>
+            </properties>
+        </profile>
```

Comment:
In addition, you think we should generally leave it to local-horst?
https://github.com/knative-extensions/eventing-integrations/blob/5d47d4604ee99fc272797acb53bcd6a469207e51/pom.xml#L28

I guess, in theory, I'd like no hard-coded `localhost`, but something that resolves via ENV_VAR? 

Here is a "big fan" of `$KO_DOCKER_REPO` like we do on `golang` :upside_down_face: 

/cc @christophd

Reply:
"local-horst" 🤣 

I am not tied to using this local config as a default. I can adjust the settings on my fork when testing locally. Feel free to remove and set a decent default value like quay.io or something

Reply:
I wanted to isolate changes for now, but sure it make sense to use `${env.KO_DOCKER_REPO}`. A note here, quarkus doesn't like additional namespaces in `registry` property, only `host:port` combination. So `kind.local` is fine, but not `gcr.io/knative-nightly` - our default in release script. :)

Therefore I've used whole `image` property here.